### PR TITLE
SK launch

### LINKED ISSUE #256: Checklist launch in Slovakia
See checklist further down 👇

### PR DESCRIPTION
**Pull request by sbocinec**
* Update SK translations for the launch
* Update SK wordlist
* Add SK social media image
* Other small SK related fixes

#256

### CODE REVIEW

File: app/views/pages/nl-lang-privacy-policy.ejs [hunk 1]
```diff
@@ -52,7 +52,7 @@
         <h2>Analytics</h2>
         <p>
           We gebruiken analytics om inzicht te krijgen in het gebruik van de website. We gebruiken daarvoor <a
-            href="https://simpleanaltyics.io" target="_blank">Simple Analytics</a>
```
Comment (fossecode) [inline app/views/pages/nl-lang-privacy-policy.ejs]: Good catch!

**Comment by fossecode** [comment]
I don't know slovak, but other than that it looks good! 😄Could you add an image to be used on social media when people share the site? Should be located in `/static/sk/social1.png`

**Review by fossecode — APPROVED** [review]

**Comment by fossecode** [comment]
Ready to merge? :)

**Comment by sbocinec** [comment]
> Ready to merge? :)

no no, it's an early draft yet, WIP :-) I will need to properly go through all the views in the app and I expect there are lot of mistakes and inaccurate translations. Also the privacy policy needs some tweaking, etc... So please ignore this PR until i remove the `[WIP]` prefix (i forgot to create this as a draft PR :-| ). Thank you!

**Review by fossecode — COMMENTED** [review]
Will review again when it is no longer WIP

**Comment by adriaandotcom** [comment]
Fixed your merge conflict

File: app/views/pages/nl-lang-privacy-policy.ejs [hunk 2]
```diff
@@ -63,7 +63,7 @@
 <p>
   We gebruiken analytics om inzicht te krijgen in het gebruik van de website. We
   gebruiken daarvoor
-  <a href="https://simpleanaltyics.io" class="text-cvs-red hover:underline" target="_blank">Simple Analytics</a>
+  <a href="https://simpleanalytics.io" class="text-cvs-red hover:underline" target="_blank">Simple Analytics</a>
```
Comment (adriaandotcom) [inline app/views/pages/nl-lang-privacy-policy.ejs]: It's `https://simpleanalytics.com` actually

File: app/views/pages/sk-lang-privacy-policy.ejs [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
Comment (adriaandotcom) [inline app/views/pages/sk-lang-privacy-policy.ejs]: Oops. Please copy the format of the other privacy policy pages. The header and footer did change!

**Review by adriaandotcom — CHANGES_REQUESTED** [review]

**Review by Kilian — APPROVED** [review]

**Comment by Kilian** [comment]
I'm overwriting @adriaanvanrossum review and pressing the merge button. The review has no changes requested anymore but Github didn't pick that up.

**Comment by adriaandotcom** [comment]
Perfect

**Pull request merged**